cc/crypto/hpke/recipient_context.cc

```diff
@@ -29,6 +29,9 @@
 #include "openssl/hpke.h"
 
 namespace oak::crypto {
+
+const uint64_t kStartingSequenceNumber = 0;
```

Comment:
Are we planning to change this value at some point? Just curious

Reply:
This will be removed once we finalize what protocol to use for streaming crypto